grai-server/app/the_guide/settings.py

```diff
@@ -88,7 +123,7 @@ def clean_allowed_hosts(val):
     "whitenoise.middleware.WhiteNoiseMiddleware",
     "django.contrib.sessions.middleware.SessionMiddleware",
     "django.middleware.common.CommonMiddleware",
-    "django.middleware.csrf.CsrfViewMiddleware",
+    # "django.middleware.csrf.CsrfViewMiddleware",
```

Comment:
Is this just removing CSRF?

Reply:
I'm not sure if it completely disables CSRF but the [docs](https://docs.djangoproject.com/en/4.1/ref/csrf/) describe it providing a specific form of subdomain protection and I wanted to see if that was responsible for the issue I was seeing when deployed to the server.